### ISSUE TITLE
PoissonRecon causes crash on Silicone chip Macbook

### ISSUE THREAD

**Issue by ArcSketch** [CloudCompare/CloudCompare#1830]
When using Poisson surface reconstruction with more than one or two threads CloudCompare immediately crashes with my M2 Max 32 GB MacBook Pro.

It's the same result every time no matter the file format and in both of the latest versions 2.12.3 and 2.13.alpha (Qt 5.15.2)

The only exceptions where I have managed to run PoissonRecon with up to 12 threads have been tests with clouds under 1K points, which then have finished calculating immediately (as in literally before the loading window has time to show up). So it's likely caused by some sort of memory issue.

**Comment by dgirardeau** [comment]
Yes, there have been many issues reported on macOS that seem to be related to multi-threading.... We lack the expertise and support on macOS sadly 😢
